index.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <title>My Portfolio</title>
+</head>
+<body>
+
+        <header class="header-container">
+            <h3 class="white">My Portfolio</h3>
+            <img src="./assets/icons/menu.png" alt="">
```

Comment:
- [Optional] Consider wrapping the hamburger image in a button element since the hamburger is a button and also consider wrapping the button in a nav element since the button displays the nav menu. This is best for HTML semantics.